client/landing/gutenboarding/components/domain-picker/button.tsx

```diff
@@ -13,11 +13,15 @@ import DomainPicker from './list';
 import { STORE_KEY as DOMAIN_STORE } from '../../stores/domain-suggestions';
 import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
 import { isFilledFormValue } from '../../stores/onboard/types';
+import { DomainSuggestion } from '../../stores/domain-suggestions/types'; // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need the eslint ignore here? :thinking:

Reply:
The best I can come up with is that eslint was confused. This is a type import, and the type is used, but something about our setup must throw it off 🤷‍♂ 

Becomes irrelevant with the `_.partition` change because the types are nicely inferred from input.